sql/handler.cc

```diff
@@ -505,7 +498,6 @@ int ha_finalize_handlerton(st_plugin_int *plugin)
   DBUG_RETURN(0);
 }
 
-
```

Comment:
According to our coding style there should be two empty lines between functions. So original spacing was correct.